lib/salus/report.rb

```diff
@@ -301,22 +313,49 @@ def write_report_to_file(report_file_path, report_string)
             "Cannot write file #{report_file_path} - #{e.class}: #{e.message}"
     end
 
-    def send_report(remote_uri, data, format)
-      response = Faraday.post do |req|
-        req.url remote_uri
-        req.headers['Content-Type'] = CONTENT_TYPE_FOR_FORMAT[format]
-        req.headers['X-Scanner'] = x_scanner_type(format)
-        req.body = data
+    def report_headers_h(headers, headers_env_var, format)
```

Comment:
Might be a win to move this to a dedicated class.  report.rb is growing rather large and ripe for some refactoring.

Reply:
Refactored logic in report.rb specific to creating an HTTP request i.e the header and faraday to another class ReportRequest